### ISSUE TITLE
Advanced configuration described in "Relaying For Other Machines" document does not work

### ISSUE THREAD

**Issue by semente** [mail-in-a-box/mailinabox#480]
The instructions described in https://mailinabox.email/advanced-configuration.html does not work. I get the following error:

```
Jul  4 04:08:59 shomehost postfix/smtp[11987]: C081042B21: to=<user@example.net>, relay=box.example.net[1.2.3.4]:587, delay=127, delays=126/0.05/0.32/0.04, dsn=5.7.1, status=bounced (host box.example.net[1.2.3.4] said: 553 5.7.1 <root@somehost.example.net>: Sender address rejected: not owned by user relay@example.net (in reply to RCPT TO command))
```

Perhaps because this commit 13958ba4df5ae5d2d3b368ff69681c2b5002a05c

**Comment by dhpiggott** [comment]
Sorry about this - I wrote those instructions and have been meaning to open a PR updating them since 13958ba was merged. Unfortunately I only seem to find time at weekends at the moment. I'd hoped to get it updated before 13958ba was included in any tagged releases but @JoshData has released things at an impressive velocity in the last ~week!

Anyway. I'll update it soon now. In the meantime you have two options:
1. Add a regular alias forwarding root@somehost.example.net to relay@example.net.
2. Add a catch-all alias forwarding @somehost.example.net to relay@example.net.

The downside to (1) is that you need to create regular aliases for every from address that your relay account will send as.
The downside to (2) is that it may/will result in @somehost.example.net catching a lot of spam, with it accumulating in the mailbox of relay@example.net.

If/when I resolve the conflicts in #460 and if/when it is merged, MIAB would gain support for "outbound-only" aliases, allowing a variant of option (2) to be taken that wouldn't risk catching spam.

**Comment by semente** [comment]
hey @dhpiggott, thank you! I could back on issue this only today. The #460 would be really great, but the option 1 is fine for me right now.

**Comment by dhpiggott** [comment]
#460 was merged a few days ago so you can switch to using it now if you want (it's included in 0.13.0, which was just tagged).

**Comment by semente** [comment]
Great!!
